tests/test_api.py

```diff
@@ -37,7 +38,7 @@
 
 
 @pytest.fixture
-def config(root="/tmp/gitman-shared"):
+def config(root=os.path.normpath(tempfile.gettempdir() + "/gitman-shared")):
```

Comment:
If you are ok, I will let you do this, when the PR is ready. I dont know enough how pytest and fixture work.